clients/iceberg-views/src/main/java/org/apache/iceberg/nessie/NessieExtCatalog.java

```diff
@@ -104,11 +105,12 @@ public void setConf(Configuration conf) {
   }
 
   public void refresh() throws NessieNotFoundException {
-    tableCatalog.refresh();
+    // TODO only used in a single test
+    // tableCatalog.refresh();
```

Comment:
Not sure I understand this :thinking:

Reply:
Ah - that `refresh()` functionality is gone now (on `NessieCatalog` since https://github.com/apache/iceberg/pull/4491).
Updated the comment, but leaving it as a todo as a reminder.

/cc @nastra

Reply:
ah yeah the `refresh()` shouldn't be necessary anymore. There was an issue back then that required calling this explicitly but I fixed this with the refactoring to the `NessieClient` wrapper in Iceberg